e2e/tests/content-manager/uniqueness.spec.js

```diff
@@ -0,0 +1,110 @@
+import { test, expect } from '@playwright/test';
+import { login } from '../../utils/login';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+
+test.describe('Uniqueness', () => {
+  test.beforeEach(async ({ page }) => {
+    // Reset the DB and also specify that we are wiping all entries of the testing content type each time
```

Comment:
Are they not wiped already?

Reply:
No they're not, I found when moving between browsers the previous documents were still there and unique constraints would fail. I think this hasn't been a problem in other specs as we haven't been testing constraints or we've been deleting the entry within the user flow(?). 

Do you think this is the best way to handle this? Or should we find a way for `resetDatabaseAndImportDataFromPath` to clear all entries by default?